component-runtime-manager/src/test/java/org/talend/sdk/component/runtime/manager/reflect/parameterenricher/DependencyParameterEnricherTest.java

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (C) 2006-2022 Talend Inc. - www.talend.com
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.talend.sdk.component.runtime.manager.reflect.parameterenricher;
+
+import java.lang.reflect.Field;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.talend.sdk.component.api.configuration.dependency.ConnectorReference;
+
+class DependencyParameterEnricherTest {
+
+    private final DependencyParameterEnricher enricher = new DependencyParameterEnricher();
+
+    @Test
+    void onParameterAnnotation() throws ReflectiveOperationException {
+        final Field field = MyConfig.class.getDeclaredField("reference");
+        final Map<String, String> value = enricher.onParameterAnnotation("value", field.getGenericType(), null);
+        Assertions.assertNotNull(value);
+        Assertions.assertNotNull(value.get("tcomp::dependencies::connector"));
```

Comment:
Maybe test value w/ assertEquals?

Reply:
Value doesn't matter, we just need (and use) presence/absence of tag "tcomp::dependencies::connector"